citus_dev/citus_dev

```diff
@@ -44,21 +46,33 @@ def run(command, *args, **kwargs):
     return result
 
 
-def createNodeCommands(clustername, role, index=None, usessl=False):
+def createNodeCommands(clustername, role, port, arguments, index=None):
     nodename = role
     if index != None:
         nodename += "%d" % index
 
-    run(f"initdb -D {clustername}/{nodename}")
+    if arguments['--fsync']:
+        run(f"initdb -D {clustername}/{nodename} --no-sync")
+    else:
+        run(f"initdb -D {clustername}/{nodename}")
 
-    shared_preload_libraries = ['citus', 'pg_stat_statements']
+    if arguments['--no-lib']:
+        shared_preload_libraries = ['pg_stat_statements']
+    else:
+        shared_preload_libraries = ['citus', 'pg_stat_statements']
```

Comment:
This works for now, but it becomes more cumbersome if we need more preload libraries in the future.
```suggestion
    shared_preload_libraries = ['pg_stat_statements']
    
    # double negation, what does it mean?
    # we do add citus if we do _not_ provide the `--no-lib` flag
    # because citus needs to be at the beginning of the list we perform this check last, and insert `citus` at index 0, which is the beginning of the list
    if not arguments['--no-lib']:
        shared_preload_libraries.insert(0, 'citus')
```